usr/lib/common/sess_mgr.c

```diff
@@ -1206,6 +1206,46 @@ CK_RV session_mgr_set_op_state(STDLL_TokData_t *tokdata, SESSION *sess,
     return CKR_OK;
 }
 
+CK_RV session_mgr_cancel(STDLL_TokData_t *tokdata, SESSION *sess,
```

Comment:
Less than 10% of the sess_mgr.c and lock_sess_mgr.c differs. I would highly recommend to fix this as soon as possible, instead of duplicating more and more code with each new feature.
Sure, this will not be fixed in the current PR, but please add it to your ToDo list.

Reply:
100% agreed! See also issue https://github.com/opencryptoki/opencryptoki/issues/579
With that we could get rid of these duplicated files.

Reply:
ok